src/components/PopoverMenuMultiSelect/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { ListItemText } from '@mui/material';
+import { DropdownItem } from '../types';
+import Icon from '../Icon/Icon';
+import './styles.scss';
+import Popover from '../PopoverMenu/Popover';
+
+export type PopoverMenuMultiSelectProps = {
+  anchorElement: HTMLElement | null;
+  initialSelection: any[];
+  menuAlign?: 'left' | 'center' | 'right';
+  onChange: (selection: any[]) => void;
+  options: any[];
+  optionRenderer: (option: any) => string;
+  setAnchorElement: React.Dispatch<React.SetStateAction<HTMLElement | null>>;
+};
+
+export default function PopoverMenuMultiSelect(props: PopoverMenuMultiSelectProps): JSX.Element {
```

Comment:
thanks for the rename, this is more apt for sure

Reply:
I think I'll rename it to `PopoverMultiSelect`. It does not quite line up with the `PopoverMenu` component, which also renders an anchor. Rather, it is a wrapper around `Popover`